lib/components/menu/menu-links.tsx

```diff
@@ -1,25 +1,27 @@
 import React from 'react'
-import Router from 'next/router'
-import { useTheme } from 'components'
+import NextLink from 'next/link'
+import { useTheme, Avatar } from 'components'
 import Controls from 'lib/components/controls'
-import LogoIcon from 'lib/components/icons/logo'
 import { useConfigs } from 'lib/config-context'
 
 const MenuLinks = () => {
   const theme = useTheme()
   const { isChinese } = useConfigs()
-  const goHome = () => {
-    const home = isChinese ? '/zh-cn' : '/en-us'
-    Router.push(home)
-  }
 
   return (
     <nav>
-      <div className="site-name">
-        <span title={isChinese ? '回到主页' : 'Go Home'} onClick={goHome}>
-          <LogoIcon />
-        </span>
-      </div>
+      <NextLink href={isChinese ? '/zh-cn' : '/en-us'}>
+        <a className="menu__logo" aria-label="Go Home">
+          <Avatar
```

Comment:
I think it's better to use the `Image` component to determine the size of the image in advance, to avoid reflow, and to come with a skeleton.